lib/manager/jenkins/readme.md

```diff
@@ -1,13 +1,3 @@
-The Jenkins manager supports the following format of the plugin list:
+The Jenkins manager supports a custom text or yaml format of the plugin list as described [here](https://github.com/jenkinsci/plugin-installation-manager-tool#plugin-input-format). Currently, only versions from the main [update center](https://updates.jenkins.io/) are supported.
 
-```text
-plugin1:1.2.3
-plugin2:4.5 # this is a comment
-
-# this line is ignored
-
-# Renovate will not upgrade the following dependency:
-plugin3:7.8.9 # renovate:ignore
-```
-
-There's no strict specification on the name of the files, but usually it's `plugins.txt`
+There's no strict specification on the name of the files, but usually it's `plugins.txt` or `plugins.yaml`.
```

Comment:
```suggestion
There are no strict filename rules, the convention is to name the file `plugins.txt` or `plugins.yml`.
```

Maybe something like this is a bit more natural?

Reply:
Updated. I kept the file `plugins.yaml` since it is the extension used in the jenkins examples

Reply:
> I kept the file `plugins.yaml` since it is the extension used in the jenkins examples

Good call, I just fat-fingered that when I was making the suggestion...